src/datetime.js

```diff
@@ -715,7 +714,7 @@ export default class DateTime {
    * @param {string} opts.outputCalendar - the output calendar to set on the resulting DateTime instance
    * @param {string} opts.numberingSystem - the numbering system to set on the resulting DateTime instance
    * @example DateTime.fromRFC2822('25 Nov 2016 13:23:12 GMT')
-   * @example DateTime.fromRFC2822('Tue, 25 Nov 2016 13:23:12 +0600')
+   * @example DateTime.fromRFC2822('Fri, 27 Apr 2018 09:27:12 +0200')
```

Comment:
Why?

Reply:
Because '25 Nov 2016' is not a Tuesday. But I'll fix it without changing the date.

Reply:
Ah, no, that is a good reason. I don't care what the date is.